lib/base/configobject.ti

```diff
@@ -88,6 +88,7 @@ abstract class ConfigObject : ConfigObjectBase < ConfigType
 	[state, no_user_modify] double version {
 		default {{{ return 0; }}}
 	};
+	[no_user_view, no_user_modify] String identifier;
```

Comment:
"identifier"? _What kind of_ identifier?

Reply:
Isn't that obvious? You will always have to access it this ways `checkable->GetIdentifier()`, `object->GetIdentifier()` etc..,. So, we don't have to prefix it with `object` or whatever.

Reply:
But with _Icinga DB_ IMAO.